Entities/ACrab.h

```diff
@@ -515,6 +515,18 @@ int FirearmActivationDelay() const;
 	void SetLimbPathPushForce(float force);
 
 
+	/// <summary>
+	/// Gets this ACrab's Stride Sound. Ownership is not transferred!
+	/// </summary>
+	/// <returns>The SoundContainer for this ACrab's Stride Sound.</returns>
+	SoundContainer* GetStrideSound() const { return m_StrideSound; }
```

Comment:
Your `*`s are in the wrong places.
The getter should be `SoundContainer * GetStrideSound()` and the setter should be `void SetStrideSound(SoundContainer *newSound);`